src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -390,28 +426,30 @@ def continuing_criterion(carry):
         maxiter = solver_options.pop("maxiter")
 
         # Compute the 3D linear force in C[W] frame.
-        CW_f_Ci, _ = run_optimization(
+        solution, _ = run_optimization(
             init_params=init_params,
-            A=A,
-            b=b,
```

Comment:
The two kwargs `A` and `b` were placed here for enabling computing the gradient, but if it's not a priority, we can remove them for the time being

Reply:
Yep let's keep this in mind as soon as this contact model starts working reliably. There are many things I'm investigating that do not sum up (friction cone not met, possibility to apply bilateral forces, etc).